presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -2591,6 +2594,40 @@ boolean isIncreasingSequence(List<Integer> fileNames)
         return true;
     }
 
+    @DataProvider(name = "reservedWordsTestDataProvider")
+    public static Object[][] reservedWordsTestDataProvider()
+    {
+        return ReservedIdentifiers.reservedIdentifiers().stream()
+                .map(a -> {
+                    String reservedIdentifier = a.toLowerCase(Locale.ENGLISH);
+                    return new Object[] {"test_show_create_reserved_" + reservedIdentifier, reservedIdentifier};
+                })
+                .toArray(size -> new Object[size][2]);
+    }
+
+    @Test(dataProvider = "reservedWordsTestDataProvider")
+    public void testShowCreateWithReservedIdentifier(String tableName, String reservedIdentifier)
+    {
+        String createTableSql = format("" +
+                        "CREATE TABLE %s.%s.%s (\n" +
+                        "   \"" + reservedIdentifier + "\" bigint,\n" +
+                        "   c2 double,\n" +
+                        "   \"c 3\" varchar,\n" +
+                        "   \"c'4\" array(bigint),\n" +
+                        "   c5 map(bigint, varchar)\n" +
+                        ")\n" +
+                        "WITH (\n" +
```

Comment:
Would you also test columns using reserved keywords in properties like partitioned_by, bucketed_by, sorted_by, etc.?